src/main/java/io/cryostat/Cryostat.java

```diff
@@ -141,6 +143,10 @@ private void shutdown(Throwable cause) {
 
     public static void main(String[] args) throws IOException {
         final Client client = DaggerCryostat_Client.builder().build();
+        try (InputStream config = CryostatCore.class.getResourceAsStream("/logging.properties")) {
+            LogManager.getLogManager()
+                    .updateConfiguration(config, k -> ((o, n) -> o != null ? o : n));
```

Comment:
For these configurations, should we use the new configurations over the old ones if exist instead?

Reply:
Nice! Also again on this note, I notice now we just replace with all new configs. Should we merge new properties and old ones (i.e. if a new property is specified, use it. Otherwise use the default/old ones)?

Reply:
Potentially sure, but in practice this is the `main` method and the first instance of any config being loaded anyway. The `CryostatCore.initialize()` actually tries to set some defaults of its own too, so I did this first here to set values that the `-core` won't override since there are already defined values for the keys. Hard to say from this perspective what the "right" behaviour is for selecting a value in a generic way, right now we just need these specific values to be set.

Reply:
Right that makes sense! Thanks for explaining :D